tests/sqlite/builder/test_sqlite_transaction.py

```diff
@@ -48,6 +54,10 @@ def test_transaction_globally(self):
         db.begin_transaction("sqlite")
         db.rollback("sqlite")
 
+    def test_transaction_decorator(self):
+        result = create_user()
```

Comment:
Now because of how python works the way you're using it, you are technically calling the user class here.